test/__snapshots__/mathml-spec.js.snap

```diff
@@ -27,6 +27,9 @@ exports[`A MathML builder should generate the right types of nodes 1`] = `
       <mi>
         sin
       </mi>
+      <mo>
+        &amp;ApplyFunction;
```

Comment:
I think this should be `&ApplyFunction;`, right? `makeText` above must be escaping the ampersand in a way we don't want...

Reply:
I missed that. Thanks for the comment.

Reply:
Is there a reference to show that `&ApplyFunction;` is equivalent to Unicode Character 'FUNCTION APPLICATION' (U+2061) in MathML?  It seems to be, but I couldn't find one...

Reply:
This [cheat sheet](http://www.html-5.com/cheat-sheet/html-character-codes.html) gives the equivalence. You can also type in `\sin x`  into the LaTeX box at [iMathEQ](http://www.imatheq.com/imatheq/com/imatheq/math-equation-editor-latex-mathml.html) and ask it to give you their version of the equivalent MathML.

Reply:
[This page](https://www.w3.org/TR/xml-entity-names/bycodes.html) is from a better source.